Jenkinsfile

```diff
@@ -469,6 +481,19 @@ def AndroidTest(stashName) {
   }
 }
 
+def NetCoreTest(String node, String stashName) {
+  return {
+    nodeWithCleanup(node) {
+      getArchive()
+      unstash stashName
+
+      dir ("Tests/Tests.NetCore/bin/${configuration}/netcoreapp1.1") {
+        sh 'dotnet Tests.NetCore.dll'
```

Comment:
Yes, I'm taking baby steps - first make sure it runs, next - transform results and report them :)